pkg/ddc/base/metadata_sync.go

```diff
@@ -42,3 +49,22 @@ func SafeSend(ch chan MetadataSyncResult, result MetadataSyncResult) (closed boo
 	ch <- result
 	return false
 }
+
+// RecordDatasetMetrics records dataset-related metrics from the given MetadataSyncResult
+func RecordDatasetMetrics(result MetadataSyncResult, datasetNamespace, datasetName string, log logr.Logger) {
+	if len(result.UfsTotal) != 0 {
+		if ufsTotal, ignoredErr := utils.FromHumanSize(result.UfsTotal); ignoredErr == nil {
```

Comment:
ignoredErr is not very descriptive and could be changed to something like parseErr.

Reply:
Done